crates/test/src/rt/mod.rs

```diff
@@ -289,20 +297,23 @@ impl Context {
 
     /// Inform this context about runtime arguments passed to the test
     /// harness.
-    ///
-    /// Eventually this will be used to support flags, but for now it's just
-    /// used to support test filters.
     pub fn args(&mut self, args: Vec<JsValue>) {
-        // Here we want to reject all flags like `--foo` or `-f` as we don't
-        // support anything, and also we only support at most one non-flag
-        // argument as a test filter.
-        //
-        // Everything else is rejected.
         let mut filter = self.state.filter.borrow_mut();
-        for arg in args {
+        let mut skip = self.state.skip.borrow_mut();
+
+        let mut args = args.into_iter();
+
+        while let Some(arg) = args.next() {
             let arg = arg.as_string().unwrap();
             if arg == "--include-ignored" {
                 self.state.include_ignored.set(true);
+            } else if arg == "--skip" {
+                skip.push(
+                    args.next()
+                        .expect("Argument to option 'skip' missing")
+                        .as_string()
+                        .unwrap(),
+                );
```

Comment:
This doesn't quite match `cargo test`'s parsing, since `cargo test` supports `--skip=foo` as well as `--skip foo`.

Reply:
Done.